adoptopenjdk-api-v3-frontend/src/test/kotlin/net/adoptopenjdk/api/AvailableReleasesPathTest.kt

```diff
@@ -21,6 +21,7 @@ class AvailableReleasesPathTest : BaseTest() {
             ApiPersistenceFactory.get().setReleaseInfo(
                 ReleaseInfo(
                     arrayOf(8, 9, 10, 11, 12),
+                    listOf(6, 7, 9, 10, 12, 13, 14),
```

Comment:
for consistency with existing code, can we use `arrayOf` instead of `listOf`?

Reply:
Done. I was having issues making an array earlier which is why it was a list. I've tried again and this time it's worked, I guess I changed something good :D